File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.py
def solution(n):
    answer = 0
    a = 1
    b = 1
    
    for i in range(1,n):
        if n == 1 or n == 2:
            return 1
        else:
            a,b = b, a+b
        answer = a
                
        
    return a%1234567

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.swift
import Foundation

func solution(_ n: Int) -> Int {
    let mod = 1234567
    
    var a = 0
    var b = 1
    
    // a에 이전 피보나치 수 담기
    for _ in 2...n {
        let temp = (a + b) % mod
        a = b
        b = temp
    }
    
    return b
}

